.github/workflows/go.yaml

```diff
@@ -6,15 +6,32 @@ on:
   pull_request:
     branches: [ v3 ]
 jobs:
+  fmt:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Go
+      uses: actions/setup-go@v5
+      with:
+        go-version: 'stable'
+    - name: Run go fmt
+      run: go fmt ./...
+    - name: Check if working tree is dirty
+      run: |
```

Comment:
I'd like to see shell scripts be in their own files so we can lint them more easily.
Leave this one alone for now...
I have some ideas that I'll present later this week or so.